src/pybind/mgr/dashboard/controllers/summary.py

```diff
@@ -41,15 +41,46 @@ def _rbd_mirroring(self):
                 warnings += 1
         return {'warnings': warnings, 'errors': errors}
 
+    def _task_permissions(self, name):
+        result = True
+        if name == 'pool/create':
+            result = self._has_permissions(Permission.CREATE, Scope.POOL)
+        elif name == 'pool/edit':
+            result = self._has_permissions(Permission.UPDATE, Scope.POOL)
+        elif name == 'pool/delete':
+            result = self._has_permissions(Permission.DELETE, Scope.POOL)
+        elif name in [
+                'rbd/create', 'rbd/copy', 'rbd/snap/create',
+                'rbd/clone', 'rbd/trash/restore']:
+            result = self._has_permissions(Permission.CREATE, Scope.RBD_IMAGE)
+        elif name in [
+                'rbd/edit', 'rbd/snap/edit', 'rbd/flatten',
+                'rbd/snap/rollback']:
+            result = self._has_permissions(Permission.UPDATE, Scope.RBD_IMAGE)
+        elif name in [
+                'rbd/delete', 'rbd/snap/delete', 'rbd/trash/move',
+                'rbd/trash/remove', 'rbd/trash/purge']:
+            result = self._has_permissions(Permission.DELETE, Scope.RBD_IMAGE)
+        return result
+
     @Endpoint()
     def __call__(self):
         executing_t, finished_t = TaskManager.list_serializable()
+        executing_tasks = []
```

Comment:
minor nit:

```python
executing_tasks = [task for task in executing_tasks if self._task_permissions(task['name'])]

Reply:
Thanks for the tip! I have changed now the for-loops accordingly.